MFVNeutralino/test/TrackMoverHighEta/Samples.py

```diff
@@ -0,0 +1,1528 @@
+#!/usr/bin/env python
```

Comment:
I don't think we need a second Samples.py, this seems _really_ hard to keep synchronized with the main one.

Reply:
Remove it. Done.